### DISCUSSION TITLE
Hooks and filters for Elementor Pro

### DISCUSSION THREAD

**Discussion by kodomastro** [elementor/elementor#18359] [Developers Community]
Where can I find documentation of hooks and filters for Elementor Pro?

I need to edit assets path in ElementorProFrontendConfig cause I use CDN. 

For standard Elementor I’m doing something like this:

add_filter( 'elementor/frontend/assets_url', function( $arg ) { return str_replace( "domain.com", "cdn.domain.com", $arg ); } );

**Comment by byronz3d** [comment]
It's 'elementor_pro/frontend/assets_url'

**Comment by perks98** [comment]
This is Elementor's developers page with a list of their hooks: [https://developers.elementor.com/docs/hooks/php/](https://developers.elementor.com/docs/hooks/php/)
There are more that they haven't listed so if you can't find what you are looking for in that it would be best to search their plugin code manually for more hooks as not 100% of their hooks are documented either due to not wanting others to use or they haven't ever got round to it.
